File: app/src/main/java/remix/myplayer/service/Playback.java
package remix.myplayer.service;

/**
 * @ClassName
 * @Description
 * @Author Xiaoborui
 * @Date 2016/12/29 09:12
 */

public interface Playback {
    void playSelectSong(int position);
    void toggle();
    void playNext();
    void playPrevious();
    void play();
    void pause();
}

File: app/src/main/java/remix/myplayer/service/Playback.kt
package remix.myplayer.service

/**
 * @ClassName
 * @Description
 * @Author Xiaoborui
 * @Date 2016/12/29 09:12
 */

interface Playback {
    fun playSelectSong(position: Int)

    fun toggle()

    fun playNext()

    fun playPrevious()

    fun play(fadeIn: Boolean)

    fun pause(updateMediasessionOnly: Boolean)
}
